user_data/strategies/Strategy002.py

```diff
@@ -15,9 +15,12 @@
 
 from user_data.strategies.util import execute, back_test
 from user_data.strategies.config import Config
+from user_data.strategies.notifier import send_start_deliminator_message
 
 
 class Strategy002(IStrategy):
+    if Config.IS_BACKTEST:
+        send_start_deliminator_message(Config.BACKTEST_COIN ,Config.BACKTEST_MONTH_LIST[Config.BACKTEST_DATA_CLEANER_MONTH_INDEX], Config.BACKTEST_DATA_CLEANER_YEAR)
```

Comment:
pass brain name

Reply:
Should the brain name be displayed on the channel or only on the log

Reply:
channel

Reply:
I have commented on line 9 --- line 9 is channel, you have the answer from that

Reply:
tried it this way but since it is done out side of method "self" wont work Freq_" + self.__class__.__name__